dev-scripts/openshift_deploy.sh

```diff
@@ -194,9 +214,9 @@ fi
 ## TODO we should create Che SA if it doesn't exist
 ## TODO we should check if che has admin rights before creating the role biding
 ## TODO if we are not in minishift we should fail if che SA doesn't have admin rights
-if [ "${OPENSHIFT_FLAVOR}" == "minishift" ]; then
+if [[ "${OPENSHIFT_FLAVOR}" =~ ^(minishift|oscp)$ ]]; then
   echo -n "[CHE] Setting admin role to \"che\" service account..."
-  oc login "${OPENSHIFT_ENDPOINT}" -u system:admin &> /dev/null
+  oc login "${OPENSHIFT_ENDPOINT}" --token="${OPENSHIFT_TOKEN}"
```

Comment:
This doesn't work: you need admin rights to give admin role to a service account. That's mandatory on minishift, and probably on oscp too.